packages/vuetify/src/components/VTextField/VTextField.sass

```diff
@@ -86,7 +73,7 @@
 /* region MODIFIERS */
 .v-text-field
   &--plain-underlined
-    --v-field-padding-top--plain-underlined: 6px
+    --v-field-padding-top--plain-underlined: 0px
```

Comment:
Wait why is this even here? All components using VField have `.v-text-field--plain-underlined` set, so the default `--v-field-padding-top: 6px` in VField.sass will only be hit if using VField standalone. 
Also I missed it in #14664 but this whole section should really be moved to VInput, currently it'll only work if VTextField is used (which is likely but not 100%)